src/it/no-main-artifact-1/verify.groovy

```diff
@@ -19,5 +19,5 @@
 
 def buildLog = new File ( basedir, "build.log")
 
-assert buildLog.text.contains( "The packaging plugin for this project did not assign "
+assert buildLog.text.contains( "The packaging plugin for project test did not assign "

```

Comment:
There are two similar IT cases in m-d-p that break once this change is applied, as they actually test the output of this plugin. Should I modify the assertion there as well?